frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -1281,6 +1281,15 @@ export const surveyLogic = kea<surveyLogicType>([
                             question: !question.question && 'Please enter a question label.',
                         }
 
+                        if (question.type === SurveyQuestionType.Link) {
```

Comment:
> they must recreate their work from scratch after encountering this type of error.

the `link` question is an example; I wonder if there are more validation types missing or if it was just this one.
Also see https://github.com/PostHog/posthog/issues/18614

Reply:
yeah I tested a bunch, but didn't see any other validations that failed silently on the BE instead of having it validated in the FE.

for https://github.com/PostHog/posthog/issues/18614, we're already scrolling to the correct section, so the last part that is missing is showing all errors on top of the form, which I'll work on next